integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/ws/core/deviceinstallation/StopDeviceSteps.java

```diff
@@ -76,7 +76,7 @@ public void theStopDeviceResponseContainsSoapFault(final Map<String, String> exp
         GenericResponseSteps.verifySoapFault(expectedResult);
     }
 
-    @Then("the platform buffers a stop device response message for device \"([^\"]*)\"")
+    @Then("the platform buffers a stop device response message for device {string}")
```

Comment:
The replaced text was a regex match for the device identification.
Since the string did not start with ^/end with $ the test is treated as a Cucumber Expression.
I chose to replace the regex with the Cucumber expression alternative {string} since it looks simpler and does the same thing.